data/tlg0007/tlg079/tlg0007.tlg079.perseus-eng3.xml

```diff
@@ -0,0 +1,1597 @@
+<?xml version="1.0" encoding="UTF-8"?> 
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader xml:lang="eng">
+      <fileDesc>
+         <titleStmt>
+            <title xml:lang="lat">Septem sapientium convivium</title>
```

Comment:
@ahanhardt 
You don't need to add multiple titles to the file itself — we have the cts work file to resolve the canonical and alternate names as well as the catalog so we need only the title as it appears in the book itself. I do appreciate the intent, though.